src/utils/validation.ts

```diff
@@ -148,7 +147,8 @@ export function validateIRelayerMakerOrder(order: IRelayerMakerOrder) {
   if (bigNumPercentageOdds.gte(FRACTION_DENOMINATOR)) {
     return `percentageOdds must be less than ${FRACTION_DENOMINATOR.toString()}`;
   }
-  if (moment.unix(parseInt(expiry, 10)).isBefore(moment())) {
```

Comment:
> moment.unix(parseInt(undefined)).isBefore(moment())
false

This check failed to detect undefined/invalid/empty `expiry`, as it's written in a fail-open way.

I preserved that behavior, but I really think that `expiry` should be validated there so that `parseInt(expiry, 10)` is a valid integer.

WDYT?

Reply:
A fix for this would be to write a fail-closed statement:
```js
  if (!(parseInt(expiry, 10) >= Date.now() / 1000)) {
    return "expiry before current time.";
  }
```